src/Command/AbstractCommand.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * This file is part of the prooph/micro.
+ * (c) 2017-2017 prooph software GmbH <contact@prooph.de>
+ * (c) 2017-2017 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\Micro\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Yaml\Yaml;
+
+abstract class AbstractCommand extends Command
+{
+    protected function getRootDir(): string
```

Comment:
We need the location for the docker-compose.yml file, or the project root dir? Why is the project root dir important?